.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+* [ ] Have you followed our guidelines in CONTRIBUTING.md? 
+* [ ] Have you self-reviewed your code and corrected any misspellings? 
+* [ ] Have you written documentation that is easy to understand?
+* [ ] Have you written descriptive commit messages? 
+* [ ] Have you added NumPy docstrings for newly added functions? 
+* [ ] Have you formatted your code using black? 
+* [ ] If you have introduced a new functionality, have you added adequate unit tests?
```

Comment:
Should we ask people to generate a coverage report? I guess this goes back to #94 , though.

Reply:
I think that is a good idea as the testing part really is in the end the most important of all these checks

Reply:
I'm going to suggest leaving user-generated coverage reports out for now. I think we should be able to do it automatically with #94.

Reply:
OK, thanks for the update! I was actually about to raise the question if it is not better to have automated code coverage reports instead of asking the users, so I agree with you -  we will leave it here.